structur3d_dap100silicone.xml.fdm_material

```diff
@@ -0,0 +1,76 @@
+  <metadata>
+    <name>
+      <brand>Structur3d</brand>
+      <material>Silicone</material>
+      <color>Clear</color>
+      <label>DAP 100% Silicone</label>
+    </name>
+    <definition>fdmprinter</definition>
+    <adhesion_info>Print on wax paper. Use tape to affix wax paper to bed.</adhesion_info>
+    <color_code>#057ec7</color_code>
+    <version>12</version>
+    <compatible>True</compatible>
+    <GUID>9bce7bd3-53fd-467a-8b92-0d70720683d9</GUID>
+    <description>Safe and reliable silicone printing. DAP 100% Silicone is ideal for the printing of parts and prototypes with flexibility, compressibility, as well as chemical and temperature resistance. Cure time is several minutes to hours.</description>
+  </metadata>
+  <properties>
+    <diameter>3.175</diameter>
+    <density>1</density>
+  </properties>
+  <settings>
+    <setting key="adhesion tendency">0</setting>
+    <setting key="retraction amount">0.0</setting>
+    <setting key="surface energy">100</setting>
+    <setting key="print temperature">0.0</setting>
+    <setting key="standby temperature">0.0</setting>
+    <setting key="heated bed temperature">0.0</setting>
+    <setting key="print cooling">0.0</setting>
+    <machine>
+      <machine_identifier manufacturer="Structur3d.io" product="Discov3ry Complete (Ultimaker 2+)" />
+      <setting key="standby temperature">0</setting>
+      <setting key="print temperature">0</setting>
+      <setting key="heated bed temperature">0</setting>
+      <hotend id="0.20mm (Clear)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="0.25mm (Red)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="0.41mm (Blue)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="0.58mm (Pink)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="0.84mm (Green)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="1.19mm (Grey)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+      <hotend id="1.60mm (Olive)">
+        <setting key="hardware compatible">yes</setting>
+      </hotend>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 2+" />
+      <setting key="standby temperature">0</setting>
+      <setting key="print temperature">0</setting>
+      <setting key="heated bed temperature">0</setting>
+      <hotend id="0.4 mm">
+        <setting key="hardware compatible">no</setting>
+      </hotend>
+      <hotend id="0.6 mm">
+        <setting key="hardware compatible">no</setting>
+      </hotend>
+      <hotend id="0.25 mm">
+        <setting key="hardware compatible">no</setting>
+      </hotend>
+      <hotend id="0.8 mm">
+        <setting key="hardware compatible">no</setting>
+      </hotend>
```

Comment:
I think we'd better make this the default for all printers except the Structur3d.io printer. Currently this material shows up for all extruders that accept ~3mm filament.

I'll make a PR to your fork to suggest a change and allow you to review that.

Reply:
Good idea. Thanks.

Reply:
See https://github.com/afinkle/fdm_materials/pull/1

We also found a bug in Cura because of this. So thanks :)

Reply:
@Ghostkeeper Thanks for all the help!